shaders/src/shading_model_standard.fs

```diff
@@ -50,7 +50,12 @@ vec3 anisotropicLobe(const PixelParams pixel, const Light light, const vec3 h,
     float ab = max(pixel.roughness * (1.0 - pixel.anisotropy), MIN_ROUGHNESS);
 
     // specular anisotropic BRDF
-    float D = distributionAnisotropic(at, ab, ToH, BoH, NoH);
+    float D = 0.0;    
+    if (!material.useWard) {
```

Comment:
are there any perf concerns here due to branching?

Reply:
Yes, these absolutely have to be moved to compile time but that requires the shader codebase consolidation ticket first (https://shapr3d.atlassian.net/browse/GFX-825?atlOrigin=eyJpIjoiMDczNTk5NmQzYTc2NGI2Njg2NjFlN2Y3NjYxNjljNTAiLCJwIjoiaiJ9). I was tempted to make it at least a coherent branch via a uniform variable but that is considerably more cumbersome than the current solution, as it would require touchups on Filament UBOs.

Reply:
makes sense, thanks